ambry-tools/src/main/java/com.github.ambry/store/DiskReformatter.java

```diff
@@ -158,10 +159,11 @@ public static void main(String[] args) throws Exception {
     DiskReformatterConfig config = new DiskReformatterConfig(properties);
     StoreConfig storeConfig = new StoreConfig(properties);
     ClusterMapConfig clusterMapConfig = new ClusterMapConfig(properties);
+    HelixPropertyStoreConfig helixPropertyStoreConfig = new HelixPropertyStoreConfig(properties);
     ServerConfig serverConfig = new ServerConfig(properties);
     ClusterAgentsFactory clusterAgentsFactory =
-        Utils.getObj(clusterMapConfig.clusterMapClusterAgentsFactory, clusterMapConfig, config.hardwareLayoutFilePath,
-            config.partitionLayoutFilePath);
+        Utils.getObj(clusterMapConfig.clusterMapClusterAgentsFactory, clusterMapConfig, helixPropertyStoreConfig,
```

Comment:
I think this change should be reverted since you did not end up changing the `ClusterAgentsFactory` constructor arguments